src/firefly/js/visualize/saga/CoverageWatcher.js

```diff
@@ -111,7 +111,7 @@ export function* watchCoverage({viewerId, options= {}}) {
 
         switch (action.type) {
 
-            case TABLE_NEW:
+            case TABLE_LOADED:
                 if (!getTableInGroup(tbl_id)) continue;
                 decimatedTables[tbl_id]= null;
                 displayedTableId = updateCoverage(tbl_id, viewerId, decimatedTables, options);
```

Comment:
yep..  added logic to skip updateCoverage when table is reloaded due to sorting.